### DISCUSSION TITLE
[solved] Antenna location

### DISCUSSION THREAD

**Discussion by a020088** [nzymedefense/nzyme#675] [Questions]
Hi I am new to Nzyme but have it installed properly. I used 4 antennas which share port tracking (2 for 2,5 and 2 for 5).

My question is where to put the antennas. Any recommendation?

**Comment by lennartkoopmann** [comment]
That will heavily depend on the physical environment you are operating in. I generally suggest to keep it free of obstructions. Mount them under the ceiling or on a shelf. Basically anywhere you'd put an access point.
